frontend/src/components/LandingPage.vue

```diff
@@ -390,6 +390,7 @@ export default {
     ...mapState(useApplicationStore, [
       'latestProgramYearId',
       'applicationIds',
+      'applicationMap',
```

Comment:
Was there any reason for moving this state entry? I was thinking for sorting but everything else is unsorted.